tests/__tests__/apiTests/articleTest.js

```diff
@@ -0,0 +1,90 @@
+import request from "supertest";
+import app from "../../../app/app.js";
+import Connection from "../../../app/db/connection.js";
+import logTestSuite from "../../util.js";
+import tokens from "../../testData/tokenTestData.js";
+import {
+  validArticleStatusUpdate,
+  invalidArticleStatusUpdate,
+  validAuthorsUpdate,
+  emptyAuthorsUpdate,
+  invalidAuthorsUpdate,
+  validArticleSlug,
+  invalidArticleSlug,
```

Comment:
This is unused, but you're right I did forget to add this to the ticket specs. Another ticket to add this validation has been created, #231